### ISSUE TITLE
No edit menu on Hyprland when monitor is offset

### ISSUE THREAD

**Issue by bekaertruben** [flameshot-org/flameshot#3641]
### Flameshot Version

v12.1.0, (this has been happening since long before)

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Arch Linux

### Description

Because I want to have my laptop screen be "below" an external monitor, I have the following rule in my Hyprland config:
`monitor=eDP-1,-1920x-1080@60,0x1440,1`
This offsets the location of this monitor so that it is below the external monitor, which is at 0x0.

However, this means that when I want to take a screenshot, the edit menu does not show up. (I assume it is trying to render on the wrong display, but I can not test this currently).

### Steps to reproduce

1. Set a monitor offset in the hyprland config
2. run flameshot gui

### Screenshots or screen recordings

![What the screen looks like](https://github.com/flameshot-org/flameshot/assets/43539018/dd301b78-de96-4c8e-9433-40448356559f)
![console log](https://github.com/flameshot-org/flameshot/assets/43539018/77357e94-a7db-4024-b012-90e78ab4b92d)

### System Information

Arch linux, fully up to date, with hyprland v0.41.0.
Monitor setup:
```
monitor=eDP-1,-1920x-1080@60,0x1440,1
monitor=HDMI-A-1,preferred,0x0,1
```

**Comment by bekaertruben** [comment]
My main laptop screen is 1080p.
The issue occurs for `monitor=eDP-1,-1920x-1080@60,0x733,1`, but not for `monitor=eDP-1,-1920x-1080@60,0x732,1` or lower.
Seems like an arbitrary value to me, perhaps that helps in figuring out what's going on

**Comment by mmahmoudian** [comment]
Since you are on Arch, would you please try the AUR. It will build Flameshot against the HEAD of the master branch which contains our latest fixes.

**Comment by bekaertruben** [comment]
> Since you are on Arch, would you please try the AUR. It will build Flameshot against the HEAD of the master branch which contains our latest fixes.

Of course, I've now moved to `flameshot-git`, and can confirm the issue still occurs.

**Comment by bekaertruben** [comment]
Another useful bit of information is that the
